### ISSUE TITLE
A11y_AzureToolsStorageExplorer_ActivitiesTab_ScreenReader: Unnecessary Table is used for the content present under "Activities" tab.

### ISSUE THREAD

**Issue by Suraj1200499** [microsoft/AzureStorageExplorer#2091]
GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1; #NVDA; #JAWS; #Win32; 

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.10.0
Windows10

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Navigate to "Activities" Tab present on right bottom of the screen and hit enter.
6. Start NVDA.
7. Navigate using Tab key to content present under "Activities" Tab, verify NVDA unnecessary announces "rows and columns" for the contents.

**​Actual:​**
Unnecessary Table is used for the contents present under "Activities" Tab because of Table used NVDA Unnecessary announces "rows and columns" for the contents.

Same issue is observed in "**JAWS**" also.

**Same issue is observed in below Scenarios:**
1. **Table** 
* Here in this scenario for Search box screen reader unnecessary announces the rows and columns.
This same issue is observed for all the Search box present under Table Section.

**Expected:**
Table should not be used for the contents present under "Activities" Tab and also NVDA should not announce "rows and columns" for the contents present under "Activities" Tab

**UserImpact:**
Screen reader users gets confusion if screen reader unnecessary announces "rows and columns" for the content present under "Activities" tab.

**Comment by craxal** [comment]
@Suraj1200499 Unable to reproduce with NVDA v2019.2.1. I don't hear any table rows or columns announced. I assume the Activities tab was empty of entries?

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not you can still repro this bug in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If it still repro's, please comment with more specific repro instructions.

**Comment by vinodbk197** [comment]
@MRayermannMSFT 
Verified this issue on:Azure Tools Storage Explorer (Version: 1.12.0)
The issue is still  reproduces, Hence reopening this bug.

**Comment by craxal** [comment]
@vinodbk197 Whether or not something _looks_ like a table is beside the point. The activities tab content _is_ a table, and the presentation as a table is intentional.

That being said, there are two potential fixes:

- Clean up the code to more strictly follow proper table semantics and add keyboard functionality that allows the user to navigate activities like a table.
   - Pros: Navigating activities will be much easier for users.
   - Cons: Relatively expensive to add and test keyboard functionality.
- Replace the table semantics with a similar grid layout.
   - Pros: Relatively cheap to implement, as it's closer to our current implementation. Allows us to use screen space more efficiently.
   - Cons: Users have to tab through every activity to get to the ones they're interested in.

Either fix isn't trivial. I'd lean more towards using a grid layout, as it's the closest to our current behavior.

**Comment by Suraj1200499** [comment]
**Actual**: Unnecessary Table is used for the contents present under "Activities" Tab because of Table used NVDA Unnecessary announces "rows and columns" for the contents.

As per above comments The activities tab content is a table, and the presentation as a table is intentional.

@Alexandria-R Need your inputs for this issue.

**Comment by craxal** [comment]
@Suraj1200499 We decided to remove the table semantics from Activities Log, as implementing the keyboard functionality is too costly at the moment. We also removed tab stops from unactionable items, so you will need to enable the appropriate reading mode to read the activity messages after it has been announced.

**Comment by Suraj1200499** [comment]
@MRayermannMSFT  Verified this issue in Private build of Windows.
Application version: 1.12.0.
This issue is no longer reproduced. Hence Closing the bug.
Added Closed Video in [One Drive](https://microsoft-my.sharepoint.com/personal/v-nisaho_microsoft_com/_layouts/15/onedrive.aspx?id=%2Fpersonal%2Fv%2Dnisaho%5Fmicrosoft%5Fcom%2FDocuments%2FStorageExplorer%5FReprovideos&originalPath=aHR0cHM6Ly9taWNyb3NvZnQtbXkuc2hhcmVwb2ludC5jb20vOmY6L3Avdi1uaXNhaG8vRXNTVUtnZm9PSzlIbUVMOFBxcVhkOTRCVXRBVkJ6S0pldHB5SDZwS0FzVXh5dz9ydGltZT1GRkRuQ25QRzEwZw) for reference.

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Win32-Sep2019;
